frontend/src/components/housing/submission/SubmissionIntakeForm.vue

```diff
@@ -536,37 +535,79 @@ onBeforeMount(async () => {
         name="activityId"
       />
 
-      <Stepper
-        v-model:activeStep="activeStep"
-        @update:active-step="onStepChange"
-      >
-        <!--
-      Contact Information
-      -->
-        <StepperPanel>
-          <template #header="{ index, clickCallback }">
+      <Stepper :value="activeStep">
+        <StepList>
+          <Step :value="0">
             <StepperHeader
-              :index="index"
+              :index="0"
               :active-step="activeStep"
-              :click-callback="clickCallback"
+              :click-callback="() => (activeStep = 0)"
```

Comment:
Assuming PrimeVue removed those useful slots so we dont have to manually track all this now?

Reply:
Yes, the top level `<Stepper :value="activeStep">` component tracks the step through the `value` prop.